prime-router/src/test/kotlin/fhirengine/engine/FhirTranslatorTests.kt

```diff
@@ -211,7 +213,7 @@ class FhirTranslatorTests {
         val terser = Terser(hl7Message)
 
         // assert
-        assert(terser.get("MSH-11-1") == "T")
+        assertThat(terser.get("MSH-11-1")).isEqualTo("T")
```

Comment:
Note: I was getting an assertion error, but it would not say why. Using assertThat makes it output the why.